---
title: La création d’une branche en dehors d’un ’Finally’ n’est pas valide
ms.date: 07/20/2015
f1_keywords:
- bc30101
- vbc30101
helpviewer_keywords:
- BC30101
ms.assetid: 16a0dc29-3657-4373-b77f-38f3cb80e6c9
ms.openlocfilehash: 72f73107b709f0b368edbd2f924c120b32d20ffd
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: HT
ms.contentlocale: fr-FR
ms.lasthandoff: 04/23/2019
ms.locfileid: "61935705"
---
# <a name="branching-out-of-a-finally-is-not-valid"></a>La création d’une branche en dehors d’un ’Finally’ n’est pas valide
Une instruction `GoTo` comprise dans un bloc `Finally` crée une branche en dehors du bloc. Il n’est pas possible de créer une branche à l’intérieur ou à l’extérieur d’un bloc `Catch` ou `Finally` .  
  
 **ID d’erreur :** BC30101  
  
## <a name="to-correct-this-error"></a>Pour corriger cette erreur  
  
- Supprimez l’instruction `GoTo` , puis implémentez la logique du programme avec des structures de contrôle de boucle ou de décision.  
  
## <a name="see-also"></a>Voir aussi

- [Try...Catch...Finally (instruction)](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
- [GoTo (instruction)](../../visual-basic/language-reference/statements/goto-statement.md)
- [Flux de contrôle](../../visual-basic/programming-guide/language-features/control-flow/index.md)
